src/Magnum/Math/Quaternion.h

```diff
@@ -399,6 +399,13 @@ template<class T> class Quaternion {
          */
         Matrix3x3<T> toMatrix() const;
 
+        /**
+         * @brief Convert to an euler vector
+         *
+         * @see @ref rotation(), @ref DualQuaternion::rotation()
+         */
+        Vector3<T> toRotation() const;
```

Comment:
`toEuler()` instead? So it's clear that you're going to the not-recommended API areas :) Can it return `Vector3<Rad<T>>`? Would be good to mention in which order the returned values are, in which order they should be combined and also a snippet that shows how to convert the Euler angles back:

```cpp
Rad x, y, z;
Quaternion q = /* I hope this is correct, please check! */
    Quaternion::rotate(z, Vector3::zAxis())*
    Quaternion::rotate(y, Vector3::yAxis())*
    Quaternion::rotate(x, Vector3::xAxis());
```

To avoid syntax error in the documentation, the docs have compiled snippets, so instead of placing it directly here what would be great is to extend `doc/snippets/MagnumMath.cpp` with a block delimited by `/* [Quaternion-toRotation] */` and reference it via `@snippet MagnumMath.cpp Quaternion-toRotation` (check other places, e.g. in Matrix4.h, to see how it's usually done). Of course if you don't feel like doing that, I can do that post-merge (but that'll delay the merge, of course ... I'm a bit tight with time lately).